### PR TITLE
Initializing dibbs-resources

### LINKED ISSUE #2019: Address eCR Viewer Build failure
Currently the eCR Viewer is failing to build during the release process. Depending on the order that building blocks get scheduled for builds this can block the release of other DIBBs services.

### PR DESCRIPTION
**Pull request by robertandremitchell**
# PULL REQUEST

## Summary
This decouples some existing work from ecr-viewer to focus on creating the image first, so that the ecr-viewer can more easily access its image rather than relying on file-pathing to find dibbs-resources.

If merged, we would also close out the related PR (https://github.com/CDCgov/phdi/pull/2035); this PR basically just relocates the dibbs-resources work here and adds the necessary GitHub Actions. 

## Related Issue
Fixes #2019

## Additional Information
I believe in order to merge this and eventually cut a release, we may need to either temporarily remove ecr-viewer from the next release so that the symlink issue will not block a release from being cut.

So sequence of follow-up PRs:
1. Add this image to phdi-charts
2. Add the reference to that chart to phdi-playground.
3. Temporarily remove ecr-viewer.
4. Cut a new release.
5. Use new image to re-uptake work to get dibbs-resources integrated with ecr-viewer.

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: .github/workflows/container-dibbs-resources.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Lint javascript for ${{env.CONTAINER}}
+        working-directory: ./containers/${{env.CONTAINER}}
+        run: |
+          npm ci
+        # run: |
+        #   npm ci
+        #   npm run lint
```
Comment (m-goggins) [inline .github/workflows/container-dibbs-resources.yaml:39]: Is it okay to remove the commented out sections?

File: .github/workflows/container-dibbs-resources.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        uses: actions/checkout@v4
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{env.NODE_VERSION}}
+      - name: Install dependencies
+        working-directory: ./containers/${{env.CONTAINER}} # Navigate to your Node.js app directory
+        run: npm install
+      # - name: Run tests
+      #   working-directory: ./containers/${{env.CONTAINER}} # Navigate to your Node.js app directory
```
Comment (m-goggins) [inline .github/workflows/container-dibbs-resources.yaml:53]: Ditto removing this?

**Review by m-goggins — COMMENTED** [review]
Looks good from what I can see! I appreciate you laying out next steps. 

One other thing we need to do is add the new container to codecov, [here](https://github.com/CDCgov/phdi/blob/main/.github/codecov.yml).

File: .github/workflows/container-dibbs-resources.yaml (hunk 1 above)
Comment (robertandremitchell) [reply .github/workflows/container-dibbs-resources.yaml]: makes sense. I mostly left them in to remember to re-add the necessary parts 😅  but i should probably do what is proper and actually just make follow-up tickets to ensure we don't forget.

File: .github/workflows/container-dibbs-resources.yaml (hunk 2 above)
Comment (robertandremitchell) [reply .github/workflows/container-dibbs-resources.yaml]: removed!

**Comment by robertandremitchell** [comment]
@DanielPaseltiner , I was looking at `phdi-charts`, and I noticed `dibbs` didn't actually have an image. So re: steps 2-3, I don't think we need to cut an image of this, which makes sense, because neither `dibbs` or `dibbs-resources` is an image we need to make public. Is that thinking correct?

If so, we might be able to merge, then move to just getting this incorporated into ecr-viewer as an image?

cc @nickbristow @BobanL

**Comment by DanPaseltiner** [comment]
> @DanielPaseltiner , I was looking at `phdi-charts`, and I noticed `dibbs` didn't actually have an image. So re: steps 2-3, I don't think we need to cut an image of this, which makes sense, because neither `dibbs` or `dibbs-resources` is an image we need to make public. Is that thinking correct?
> 
> If so, we might be able to merge, then move to just getting this incorporated into ecr-viewer as an image?
> 
> cc @nickbristow @BobanL

We certainly don't need a helm chart for either of these images, but if you look [here](https://github.com/CDCgov/phdi/pkgs/container/phdi%2Fdibbs) we do need to release them to GHCR. This what makes the image available to use as a starting point for building the other building blocks.

By "remove eCR Viewer" we are just referring to removing it from the release workflow until the build issue is resolved, right?

**Comment by robertandremitchell** [comment]
> > @DanielPaseltiner , I was looking at `phdi-charts`, and I noticed `dibbs` didn't actually have an image. So re: steps 2-3, I don't think we need to cut an image of this, which makes sense, because neither `dibbs` or `dibbs-resources` is an image we need to make public. Is that thinking correct?
> > If so, we might be able to merge, then move to just getting this incorporated into ecr-viewer as an image?
> > cc @nickbristow @BobanL
> 
> We certainly don't need a helm chart for either of these images, but if you look [here](https://github.com/CDCgov/phdi/pkgs/container/phdi%2Fdibbs) we do need to release them to GHCR. This what makes the image available to use as a starting point for building the other building blocks.
> 
> By "remove eCR Viewer" we are just referring to removing it from the release workflow until the build issue is resolved, right?

Ah okay, yeah, that makes sense. I can work on sister PRs then to add the image to phdi-charts and phdi-playground.

Correct, sorry, yeah, this would just be another PR to remove ecr-viewer from release workflow temporarily.

**Review by bamader — APPROVED** [review]
Was a bear to get through because I wanted to make sure I didn't miss anything and that I understood what was going on, but I think everything looks good! I think once we remove the ecr viewer from the release process (until symlink is handled) and then get the new dibbs-resources pushed to GHCR, we'll be good to go. So in that spirit, I think this is probably merge-ready (at least in my eyes).

**Pull request closed without merging**